packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -1,14 +1,9 @@
 import type { ReactElement } from 'react';
 import React from 'react';
-import type { AmountTypeProps, Currency } from './amountTokens';
-import {
-  normalAmountSizes,
-  getCurrencyAbbreviations,
-  currencyIndicatorMapping,
-  subtleFontSizes,
-  amountLineHeights,
-  currencyPositionMapping,
-} from './amountTokens';
+import type { CurrencyCodeType } from '@razorpay/i18nify-js';
+import { formatNumber, formatNumberByParts } from '@razorpay/i18nify-js';
```

Comment:
This file is common for both native & web, so any package imported has to be compatible on both platforms.

Reply:
yep, It's compatible.